code/game/machinery/machinery.dm

```diff
@@ -366,7 +366,7 @@ Class Procs:
 		return 1
 	return 0
 
-/obj/machinery/proc/default_unfasten_wrench(mob/user, obj/item/weapon/wrench/W, time = 20)
+/obj/proc/default_unfasten_wrench(mob/user, obj/item/weapon/wrench/W, time = 20)

```

Comment:
here's why you're wrong about this being stupid:

default_unfasten_wrench() is only called on objects that have a call to them(such as machinery)

so not everything has it